juju/juju
# mongo/mongometrics: one package to rule them all

## Description of change

Move the existing MongoDB-related Prometheus collectors into a
single new package, and introduce a new collector that tracks
MongoDB server dials. Add unit tests for the mgo stats collector,
and change the "sent ops", "received ops" and "received docs"
metrics to counters.

## QA steps

1. juju bootstrap localhost
2. juju ssh -m controller, edit agent.conf setting MGO_STATS_ENABLED=true
3. juju ssh -m controller juju-introspect metrics
(check mongo-related metrics in output)

## Documentation changes

None.

## Bug reference

None.
